src/mod/server.mod/server.h

```diff
@@ -23,7 +23,9 @@
 #ifndef _EGG_MOD_SERVER_SERVER_H
 #define _EGG_MOD_SERVER_SERVER_H
 
-#define CAPMAX      499           /*  (512 - "CAP REQ :XXX\r\n")   */
+#define CAPMAX      499           /*  (512 - "CAP REQ :XXX\r\n")     */
+#define TAGMAX      8191          /* Max size for IRCv3 message-tags */
```

Comment:
how can TAGMAX be more than 512? I know there's a capability to allow for longer message lines but eggdrop doesn't support that

Reply:
The message buffer remains 512. The tag prefix preceding the message can be 8191. This value is only used to define buffers holding a msg tag (or a TAGMSG message)